sklearn/utils/validation.py

```diff
@@ -451,11 +470,12 @@ def check_X_y(X, y, accept_sparse=None, dtype="numeric", order=None,
     y : nd-array, list or sparse matrix
         Labels.
 
-    accept_sparse : string, list of string or None (default=None)
+    accept_sparse : string, boolean or list of string (default=False)
```

Comment:
Ah sorry I missed that, maybe the message can be changed to mention both `check_array` and `check_X_y` then.